chroma-dependencies/Makefile

```diff
@@ -21,7 +21,7 @@ cleanrepo: TARGET=cleanrepo
 requirements::
 	# these are the ones provided by EL and EPEL and our own repo
 	set -e;                                                   \
-	{ echo "../chroma-externals/netaddr-0.7.5.zip";           \
+	{ echo "https://github.com/drkjam/netaddr/archive/rel-0.7.5.tar.gz"; \
```

Comment:
I would say it is generally about readability and making it obvious that one long logical line continues over many physical lines in a single block.  When you have a single block that is staggered, particularly in groups it can make it look like they are separate blocks even though they are not and making a correction does not impact overall readability.

That is the type of alignment that I have asked to be corrected.

I think looking at the block in the above code it is clear to see that is one long block with a long line exception or two.

This kind of exception happens in many places in our code where trying to be too rigid about overly long lines makes the overall readability worse, not better.